Bot-Development/Discord-Telegram-Bots/level-4/moderationbot_AdyaArchita/README.md

```
@@ -0,0 +1,107 @@
+
+4. Run the Bot
+Once configured, you can start the bot with a single command.
+```bash
+python bot.py
+```
+Your bot is now online. To stop it, press `Ctrl+C`.
+
+## Configuration Details
+All settings are managed in the `config.json` file.
+
+       Key           Type                                      Description 
```

markdownlint-cli2 (0.18.1)

Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)